src/app/window-actions.ts

```diff
@@ -115,8 +115,10 @@ const throttledWindowChanges = throttle(async (eventName, window) => {
 const throttledWindowRestore = throttle(async () => {
   notification.moveNotificationToTop();
   const mainWebContents = windowHandler.getMainWebContents();
-  if (mainWebContents && !mainWebContents.isDestroyed()) {
-    mainWebContents.focus();
+  if (isWindowsOS) {
```

Comment:
so this will only apply for windows?

Reply:
mainWebContents.focus() is buggy on macOS and putting SDA to the background

Reply:
I see approved